### PR TITLE
[CCLEX-166]: Changed timestamp format in 'Synced at' and 'Created at' fields

### PR DESCRIPTION
**Pull request by AntonNos**
### PR Checklist

Check if done, remove if not applicable:

- [x] New feature or bug fix is mentioned in the `CHANGELOG`.

Screenshot 2022-10-03 at 12 53 42

### CODE REVIEW

**Comment by stefcameron** [comment]
👀

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # ChangeLog
 
+## UNRELEASED
+
+- Changed timestamp format in `Created at` and `Synced at` fields  in all the "View details" entity panels.
```
Comment (stefcameron) [inline CHANGELOG.md:5]: Thanks for adding the changelog entry! Just a little typo I found:

```suggestion
- Changed timestamp format in `Created at` and `Synced at` fields in all the "View details" entity panels.
```

File: src/renderer/catalogEntityDetails.js [hunk 2]
```diff
@@ -28,8 +28,8 @@ const getDateValue = (date) => {
   const dateObj = new Date(date);
   return dateObj.getTime() === 0
     ? unknownValue()
-    : `${dayjs(date).format('MMM DD, YYYY, HH:mm:ss')} (${dayjs(date).fromNow(
-        true
+    : `${dayjs(date).fromNow(true)} ago (${dayjs(date).format(
+        'YYYY-MM-DD, HH:mm:ss'
```
Comment (stefcameron) [inline src/renderer/catalogEntityDetails.js:32]:
```suggestion
        'YYYY-MM-DD HH:mm:ss'
```

I don't think we need the comma in the formatting.

**Review by stefcameron — CHANGES_REQUESTED** [review]
Thanks for fixing this!

File: CHANGELOG.md (hunk 1 above)
Comment (AntonNos) [reply CHANGELOG.md]: Fixed

File: src/renderer/catalogEntityDetails.js (hunk 2 above)
Comment (AntonNos) [reply src/renderer/catalogEntityDetails.js]: Fixed

**Review by stefcameron — APPROVED** [review]
👍

**Pull request merged**